openreview/openreview.py

```diff
@@ -1325,11 +1325,13 @@ class Group(object):
     :param details:
     :type details: optional
     """
-    def __init__(self, id, readers, writers, signatories, signatures, cdate = None, ddate = None, members = None, nonreaders = None, web = None, details = None):
+    def __init__(self, id, readers, writers, signatories, signatures, cdate = None, ddate = None, tcdate=None, tmdate=None, members = None, nonreaders = None, web = None, details = None):
```

Comment:
@melisabok Can you also add these Group attributes to the docstring.

Reply:
done